.github/workflows/CI.yml

```diff
@@ -41,6 +41,7 @@ jobs:
           python-version: 3.7
       - run: make install-for-development
       - run: make lint-minimal
+      - run: npm run lint:css
```

Comment:
Not sure if it's related but above the line

```
      - run: npm install
```

Should be

```
      - run: npm ci
```

`npm ci` will always install exactly how package lock defines it.

Reply:
Trying it

Reply:
See below. The visual regression tests. These were added more recently. My guess is make frontend does all the frontend setup

Reply:
Sorry. Make frontend starts the dev server. That's not what you want.

Reply:
Do you mean 

```
`jobs:

  build:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - id: nvm
        run: echo "##[set-output name=NVMRC;]$(cat .nvmrc)"
      - uses: actions/setup-node@v2-beta
        with:
          node-version: "${{ steps.nvm.outputs.NVMRC }}"
      - run: npm install
      - run: npm run build`

```  

Do you mean I should replace npm install present here with npm ci?

Reply:
`make setup` or make setup-frontend should be good

https://github.com/wagtail/sphinx_wagtail_theme/blob/7c48c33e0f79628a75311ff02091efce245b6b18/Makefile#L182